web/src/settings_data.ts

```diff
@@ -116,12 +106,6 @@ function user_has_permission(policy_value: number): boolean {
 }
 
 export function user_can_invite_users_by_email(): boolean {
-    if (
-        realm.realm_invite_to_realm_policy ===
-        settings_config.email_invite_to_realm_policy_values.nobody.code
-    ) {
-        return false;
```

Comment:
Removed this since we already check nobody value in has_permission.

Reply:
Yep, seems reasonable.